doc/source/user/c-info.how-to-extend.rst

```diff
@@ -622,7 +623,11 @@ updates the output array.
         if (arr1 == NULL) return NULL;
         arr2 = PyArray_FROM_OTF(arg2, NPY_DOUBLE, NPY_IN_ARRAY);
         if (arr2 == NULL) goto fail;
+    #if NPY_API_VERSION >= 0x0000000c
+        oarr = PyArray_FROM_OTF(out, NPY_DOUBLE, NPY_INOUT_ARRAY2);
```

Comment:
Should be `NPY_ARRAY_INOUT_ARRAY2`, also, above should be `NPY_ARRAY_IN_ARRAY` in both places.